dbms/src/Flash/Pipeline/Pipeline.h

```diff
@@ -90,6 +90,8 @@ class Pipeline : public std::enable_shared_from_this<Pipeline>
 
     String getFinalPlanExecId() const;
 
+    void setHasPipelineBreakerWaitTime(bool value) { has_pipeline_breaker_wait_time = value; }
```

Comment:
It seems that the type of source physical plan can be used to determine if there is a pipeline breaker time, without needing external input.

Reply:
Yes, you're right. I think it's natural to set the flag during pipeline building process also.